### DISCUSSION TITLE
Lighthouse positioning accuracy measurement

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [bitcraze/discussions#841] [Q&A]
Is there a way to measure the accuracy of the lighthouse positioning through the crazyflie?

**Comment by krichardsson** [comment]
Not a very good measurement.
I think you are using the crossing beam flavor (#839), in this case you can use the [`lighthouse.delta`](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#lighthousedelta) to get an idea of the errors in the system. In the crossing beam method we use the intersection of the light planes (called the beam as it forms a line from the center of the base station) from each base station. As there are errors in the system, the beam will most likely not cross, instead we find the point where the beams are as close as possible. The estimated position of the sensor will be the point between the beams.
The `lighthouse.delta` log contains the smallest distance between the beams. It does not really show how good the position is, but rather how much the base stations "disagree".
